<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuSolver</class>
 <widget class="QMainWindow" name="SudokuSolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SudokuSolver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate New Puzzle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="solve">
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="dificultyCombo">
       <item>
        <property name="text">
         <string>Choose Dificulty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Easy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extreme (Default)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Imposible</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ARE YOU SURE?</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="speedCombo">
       <item>
        <property name="text">
         <string>Speed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 ms/operation (Default)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 ms/operation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 ms/operation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 ms/operation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 ms/operation</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="boardLayout"/>
   </widget>
   <widget class="QLabel" name="opLbl">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Operations: </string>
    </property>
   </widget>
   <widget class="QLabel" name="counterLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
